Tests/Chairside.Tests.ps1

```diff
@@ -1,4 +1,4 @@
-describe 'Chairside System' {
+Describe 'Chairside Installation' -Tags "chairside" {
```

Comment:
How can this tag be set ?

Reply:
here: https://github.com/dentalwings/validation/pull/9/files#diff-3fb898b29d93c9fd71da24df805f5db2d434da7474eb08c61888c9e39d51974fR37

as chairside systems don't have a scannerinfo file, this can only be set manually